apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/settings/components/platform-fees.tsx

```diff
@@ -36,7 +36,7 @@ const CommonPlatformFeeSchema = z.object({
   /**
    * platform fee basis points
    */
-  platform_fee_basis_points: BasisPointsSchema.default(0),
+  platform_fee_basis_points: BasisPointsSchema,
   /**
    * platform fee recipient address
    */
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +43]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->remove default from here as well<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5342?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->